.github/workflows/website.yml

```diff
@@ -76,7 +76,8 @@ jobs:
           retention-days: 1
 
   docs-swift:
-    runs-on: ubuntu-latest
+    # ref: https://github.com/swift-actions/setup-swift/issues/506
+    runs-on: ubuntu-20.04
```

Comment:
The `docs-swift` check was failing with the error below until I made this change. It's not clear to me why this wouldn't have been an issue before, or if this change is correct.

```
Run swift-actions/setup-swift@v2
/usr/bin/gpg --import /home/runner/work/_temp/1feadda7-402d-4250-a671-d56e59bbe291
gpg: directory '/home/runner/.gnupg' created
gpg: keybox '/home/runner/.gnupg/pubring.kbx' created
gpg: no valid OpenPGP data found.
gpg: Total number processed: 0
Error: Unexpected error, unable to continue. Please report at https://github.com/swift-actions/setup-swift/issues
The process '/usr/bin/gpg' failed with exit code 2
Stacktrace:
Error: The process '/usr/bin/gpg' failed with exit code 2
    at ExecState._setResult (/home/runner/work/_actions/swift-actions/setup-swift/v2/dist/index.js:3190:25)
    at ExecState.CheckComplete (/home/runner/work/_actions/swift-actions/setup-swift/v2/dist/index.js:3173:18)
    at ChildProcess.<anonymous> (/home/runner/work/_actions/swift-actions/setup-swift/v2/dist/index.js:3067:27)
    at ChildProcess.emit (node:events:514:28)
    at maybeClose (node:internal/child_process:1105:16)
    at ChildProcess._handle.onexit (node:internal/child_process:305:5)
```

Reply:
The job appears to be flaky (https://github.com/swift-actions/setup-swift/issues/591), I think you can remove this change.

Reply:
This appears to fail consistently once removing the change